src/services/refactors/moveToNewFile.ts

```diff
@@ -800,7 +823,8 @@ function filterBindingName(name: BindingName, keep: (name: Identifier) => boolea
 
 function forEachReference(node: Node, checker: TypeChecker, onReference: (s: Symbol) => void) {
     node.forEachChild(function cb(node) {
-        if (isIdentifier(node) && !isDeclarationName(node)) {
+        const isExportSymbol = isExportSpecifier(node.parent) || isExportAssignment(node.parent);

```

Comment:
I think if what I suggested above makes sense, you also don't need this change, because you don't need to call `onReference` on export nodes. Let me know if it makes sense.

Reply:
I would have used SymbolSet instead of map, just forgot that I can simply do `(checker.getAliasedSymbol)` on exports map. Again, thanks for the suggestion.